packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -148,16 +142,12 @@ export const makeInnerVault = (
     phase = newPhase;
   };
 
-  const assertPhase = allegedPhase => {
+  const assertVaultIsOpen = () => {
     assert(
-      phase === allegedPhase,
-      X`vault must be ${allegedPhase}, not ${phase}`,
+      phase === VaultPhase.ACTIVE || phase === VaultPhase.LIQUIDATED,
```

Comment:
Good call. Since the vault context is implied I made it simply `assertCloseable`. That highlighted that some other places really needed `assertActive` so I made that too.